repos/rust-lang/crates.io.toml

```
@@ -0,0 +1,21 @@
+org = "rust-lang"
+name = "crates.io"
+description = "The Rust package registry"
```

To accommodate #1353.

```suggestion
description = "The Rust package registry"
homepage = "https://crates.io"
```